hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -478,6 +478,18 @@ public void run() {
     }
 
     @Override
+    public void reset() {
+        for (BasicInvocation invocation : invocations.values()) {
+            try {
+                invocation.notify(new MemberLeftException());
```

Comment:
Are you sure notify is a good method name since it is commonly used for wait/notify